networkx/drawing/nx_pylab.py

```diff
@@ -470,6 +477,9 @@ def draw_networkx_nodes(
         labelleft=False,
     )
 
+    if margins is not None:
+        ax.margins(*margins) if isinstance(margins, Iterable) else ax.margins(margins)
```

Comment:
I'm not sure if standards have changed, but I think python zen says we use the in-line `___ if ___ else ___` when the result produces an object (that is, when the in-line feature is what is desired).  For flow control we should use usual `if` structure...  I don't feel strongly, but what do you think?

```suggestion
        if isinstance(margins, Iterable):
            ax.margins(*margins)
        else:
            ax.margins(margins)
```

Reply:
I wasn't aware of that - sounds like a good suggestion to me :+1:

Reply:
That's an interesting point that I was not aware of either. Sounds good!